include/quicr/message_types.h

```diff
@@ -29,7 +29,7 @@ enum class MessageType : uint8_t
   PublishIntentEnd,
   Fetch,
 
-  PeerMsg=128
+  PeerMsg = 128
```

Comment:
We should be explicit for the other values as well.

Reply:
This is a hack that came later, it's the only one that specifically needs that number. The rest do not.